types/src/jsonrpc_v2/error.rs

```diff
@@ -0,0 +1,10 @@
+use thiserror::Error;
+
+/// Error.
```

Comment:
Some docs here maybe. Missing docs for all variants.

Reply:
> I wasn't in the mood on writing docs for the moved code

Yes, we need docs :)